src/olympia/reviewers/decorators.py

```diff
@@ -2,7 +2,7 @@
 
 from django.core.exceptions import PermissionDenied
 
-from olympia import amo
+from olympia.amo import permissions
```

Comment:
nit-question: should we be importing from `olympia.amo` or `olympia.constants`?

Reply:
In an ideal world, we'd stop the crazy `import *` in `amo/__init__.py` so good point, I'll use `olympia.constants`.